core/src/main/java/bisq/core/xmr/connection/persistence/model/PersistableMoneroConnection.java

```diff
@@ -0,0 +1,43 @@
+package bisq.core.xmr.connection.persistence.model;
+
+import bisq.common.proto.persistable.PersistablePayload;
+
+import protobuf.EncryptedMoneroConnection;
+
+import com.google.protobuf.ByteString;
+
+import java.net.URI;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder(toBuilder = true)
+public class PersistableMoneroConnection implements PersistablePayload {
```

Comment:
I'm thinking there should only be `UriConnection implements PersistablePayload`, just like `XmrAddressEntry`.